src/components/VolunteerDetailNotes.vue

```diff
@@ -26,45 +26,92 @@
-            <IconPhoneOutgoing v-if="note.type == 'phone outgoing'" />
-            <p class="inline pl-4" v-if="note.type == 'email'">E-Mail</p>
-            <p class="inline pl-4" v-else-if="note.type == 'note'">Notiz</p>
-            <p class="inline pl-4" v-else-if="note.type == 'phone incoming'">Telefonat</p>
-            <p class="inline pl-4" v-else-if="note.type == 'phone outgoing'">Telefonat</p>
+          <td
+            class="font-bold pl-4"
+            :class="{
+              'rounded-tl-md': index === 0,
+              'rounded-bl-md':
+                index === volunteerStore.volunteerNotes.length - 1 && !expandedRows.includes(index)
+            }"
+          >
+            <IconMail
+              class="text-vologray-600"
+              v-if="note.type == 'email' || note.type == 'E-Mail'"
+            />
+            <IconMemo v-if="note.type == 'note' || note.type == 'Notiz'" />
+            <IconPhoneIngoing
+              v-if="note.type == 'phone incoming' || note.type == 'Eingehender Anruf'"
+            />
+            <IconPhoneOutgoing
+              v-if="note.type == 'phone outgoing' || note.type == 'Ausgehender Anruf'"
+            />
+            <p class="inline pl-4" v-if="note.type == 'email' || note.type == 'E-Mail'">E-Mail</p>
+            <p class="inline pl-4" v-else-if="note.type == 'note' || note.type == 'Notiz'">Notiz</p>
+            <p
+              class="inline pl-4"
+              v-else-if="note.type == 'phone incoming' || note.type == 'Eingehender Anruf'"
+            >
+              Telefonat
+            </p>
+            <p
+              class="inline pl-4"
+              v-else-if="note.type == 'phone outgoing' || note.type == 'Ausgehender Anruf'"
+            >
+              Telefonat
+            </p>
             <p class="inline pl-4" v-else>{{ note.type }}</p>
           </td>
           <td>{{ note.user }}</td>
           <td>{{ note.timestamp.split('T').slice(0, 1).join().split('-').reverse().join('.') }}</td>
-          <td>
+          <td
+            :class="{
+              'rounded-tr-md ': index === 0,
+              'rounded-br-md':
+                index === volunteerStore.volunteerNotes.length - 1 && !expandedRows.includes(index)
+            }"
+          >
             <IconArrowShowDetailSummary
+              class="mx-4"
               :class="{ 'transform rotate-180': expandedRows.includes(index) }"
             />
           </td>
         </tr>
 
         <tr v-if="expandedRows.includes(index)" :key="note.id">
-          <td colspan="4" class="h-14 px-12 pb-4 border-b">
-            <div>
-              <p>{{ note.note }}</p>
-            </div>
+          <td colspan="3" class="h-14 pl-8 pb-4 border-b">
+            <p class="bg-vologray-100 px-4 py-2 mx-4 rounded-md">{{ note.note }}</p>
+          </td>
+          <td class="h-14 pb-4 border-b">
+            <button><IconPenEdit class="mx-4" @click="openEditModal(note)" /></button>
```

Comment:
Only show the IconPenEdit-Component if note is from loggedIn User. otherwise there will be an unhandled error if you try to edit a note from different User